ui/src/main/java/edu/wpi/grip/ui/preview/PreviewsController.java

```diff
@@ -26,20 +29,20 @@
 @Singleton
 public class PreviewsController {
 
-    @FXML private HBox previewBox;
+    @FXML
```

Comment:
Why did you change this class?  What does this have to do with the pipeline stuff?